src/System.Management.Automation/engine/Attributes.cs

```diff
@@ -1152,8 +1104,7 @@ private void ValidateRange(object element, ValidateRangeKind rangeKind)
                     "ValidationRangeElementType",
                     innerException: null,
                     Metadata.ValidateRangeElementType,
-                    element.GetType().Name,
-                    typeof(int).Name);
+                    element.GetType().Name, typeof(int).Name);
```

Comment:
```suggestion
                    element.GetType().Name,
                    typeof(int).Name);
```

Reply:
To my eye, it makes more sense to have all the arguments that are being rolled into the `params` argument on the same line, since they'll all be assigned to the same argument. 

Though I concede it does look a little inconsistent just looking at the call, without looking at the arguments that the method will see.